datalake/backend/tests/test_processing_stack.py

```diff
@@ -34,14 +35,23 @@
 
 
 @mark.infrastructure
-def test_should_create_state_machine_arn_parameter(ssm_client: SSMClient) -> None:
+def test_should_retrieve_state_machine_arn_parameter(ssm_client: SSMClient) -> None:
```

Comment:
As discussed, this test name could probably be better phrased as "state machine ARN parameter should exist". Ditto below.